HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -91,6 +90,27 @@ string custodianCode
             };
             SendEmail(emailModel);
         }
+        public void SendReferenceCodePendingLAEmail
+        (
+            string emailAddress,
+            string recipientName,
+            string referenceCode,
+            string custodianCode
+        )
```

Comment:
It's absolutely all over the place in this codebase, but the standard C# style for multiline parameters is:

```
public void SendReferenceCodeEmailForPendingLocalAuthority(
    string emailAddress,
    string recipientName,
    ReferralRequest referralRequest)
{
```

Can you make that change for your methods?

Reply:
You haven't quite made this change correctly.

The opening parenthesis `(` should be on the same line as the method name, so e.g.:
```
private void SendReferenceCodeEmail(
    string emailAddress,
    string recipientName,
    ReferralRequest referralRequest,
    ReferenceCodeConfiguration template)
```
rather than:
```
private void SendReferenceCodeEmail
(
    string emailAddress,
    string recipientName,
    ReferralRequest referralRequest,
    ReferenceCodeConfiguration template)
```